src/components/Button/Button.module.css

```diff
@@ -77,33 +77,30 @@
 .primary {
   border: none;
   border-radius: 4px;
-  background: linear-gradient(135deg, #D63FE6 0%, #A30CB3 100%);
-  box-shadow: 0 2px 4px -2px rgba(204,15,224,0.4), 0 2px 4px -2px rgba(133,129,153,0.35);
+  background: var(--primary);
 }
 
 .primary .textLabel {
-  color: #FAFAFA;
+  color: var(--darkmode-static-white);
```

Comment:
Not necessarily as part of this PR, but I think we should rename this color to just `--static-white`